server.js

```diff
@@ -1,29 +1,73 @@
 import express from "express";
 import cors from "cors";
+import getEndpoints from "express-list-endpoints";
 
 // If you're using one of our datasets, uncomment the appropriate import below
 // to get started!
 // import avocadoSalesData from "./data/avocado-sales.json";
 // import booksData from "./data/books.json";
 // import goldenGlobesData from "./data/golden-globes.json";
-// import netflixData from "./data/netflix-titles.json";
+ import netflixData from "./data/netflix-titles.json";
 // import topMusicData from "./data/top-music.json";
-
+//import members from "./data/technigo-members.json";
+import { findConfigUpwards } from "@babel/core/lib/config/files";
 // Defines the port the app will run on. Defaults to 8080, but can be overridden
 // when starting the server. Example command to overwrite PORT env variable value:
 // PORT=9000 npm start
 const port = process.env.PORT || 8080;
 const app = express();
 
 // Add middlewares to enable cors and json body parsing
-app.use(cors());
-app.use(express.json());
+app.use(cors());// decides what origins to accept. Corse() means all origins are accepted
+app.use(express.json()); //unpacks info from requests
 
 // Start defining your routes here
 app.get("/", (req, res) => {
-  res.send("Hello Technigo!");
+  res.send(getEndpoints(app));
```

Comment:
This was really smart! I wish I had done the same, will do it for this weeks project for sure!